dair_pll/inertia.py

```diff
@@ -203,14 +288,82 @@ def pi_to_theta(pi: Tensor) -> Tensor:
         return torch.cat((alpha, d_vector, s_and_t), -1)
 
     @staticmethod
-    def pi_to_drake_spatial_inertia(pi: Tensor) -> Tensor:
+    def pi_o_to_pi_cm(pi_o: Tensor) -> Tensor:
```

Comment:
originating from some of my poor prior documentation, I'm noticing that `pi_o_to_theta` can handle arbitrary dimensions of batching, whereas it's comments suggest it's only being used with 1 batching dimension. This function can also only handle 1 dimension. Suggest adding a `assert pi_o.dim() <= 2` s to make sure that the function doesn't get called with e.g. a `(*,*,10)`